scripts/mk_all.lean

```diff
@@ -34,13 +36,19 @@ def mkAllCLI (args : Parsed) : IO UInt32 := do
     let allFiles ← getAllModules git d
     let fileContent := ("\n".intercalate (allFiles.map ("import " ++ ·)).toList).push '\n'
     if !(← pathExists fileName) then
-      IO.println s!"Creating '{fileName}'"
+      if check then
+        IO.println s!"File {fileName} does not exist"
+      else
+        IO.println s!"Creating '{fileName}'"
+        IO.FS.writeFile fileName fileContent
       updates := updates + 1
-      IO.FS.writeFile fileName fileContent
     else if (← IO.FS.readFile fileName) != fileContent then
-      IO.println s!"Updating '{fileName}'"
+      if check then
+        IO.println s!"The file '{fileName}' is out of date: run `lake exe mk_all` to update it"
```

Comment:
```suggestion
        IO.println s!"The file '{fileName}' is out of date: run `lake exe mk_all{if git then " --git" else ""}` to update it"
```
(untested)

Reply:
I would prefer *not* to do this: the `--check` flag is meant to be passed in during CI only, and CI only runs on the CI output (which equals the local output). So, I'd tend to recommend the simplest thing which works, and I guess that is this one.

Reply:
... a moment later I realised that this line really does - it doesn't seem to hurt either. (It only prints --git if the user specified this manually, in which case mentioning the flag will not be confusing.) I've included it now.